templates/AssetInventory/CardView/CardView.tsx

```diff
@@ -44,7 +50,13 @@ export const CardView = ({ id, instance }: CarViewProps) => {
             statusColor={row?.original?.statusColor}
             icon={getStatusIcon(row?.original?.statusColor)}
           >
-            <HvCardHeader title={row?.values?.name} />
+            <HvCardHeader
```

Comment:
We already have the `LoadingSkeleton` component so we don't need these changes in card view to use `HvSkeleton`